stories/components/errormessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Text, IconButton, Collapse } from '../../../src'
+import Alert, { Color } from '@material-ui/lab/Alert'
```

Comment:
reuse Alert and Title from '../../../src'

and pick the color off the props e.g. AlertProps['color'] - then it will keep upto date if we make changes to our alerts